# Header tags

In [1]:
#importing modules
from bs4 import BeautifulSoup as bs
import requests
import pandas as pd

In [2]:

tags = requests.get('https://en.wikipedia.org/wiki/Main_Page')

In [3]:
tags

<Response [200]>

In [4]:
soup = bs(tags.content)

In [5]:
header=[]

for i in soup.find_all('span',class_='mw-headline'):
    header.append(i.text.replace('\xa0...',' '))

In [6]:
header

["From today's featured article",
 'Did you know ',
 'In the news',
 'On this day',
 "Today's featured picture",
 'Other areas of Wikipedia',
 "Wikipedia's sister projects",
 'Wikipedia languages']

In [7]:
df=pd.DataFrame({'Headers':header})

In [8]:
df

Headers
0  From today's featured article
1                  Did you know 
2                    In the news
3                    On this day
4       Today's featured picture
5       Other areas of Wikipedia
6    Wikipedia's sister projects
7            Wikipedia languages

# Imdb top rated movies

In [9]:
imdb=requests.get('https://www.imdb.com/chart/top/')

In [10]:
imdb

<Response [200]>

In [11]:
be=bs(imdb.content)

In [12]:
movies=[]

for movie in be.find_all('td',class_='titleColumn'):
    movie=movie.text.replace('\n','')
    movie=movie.split('|')[0]
    movies.append(movie)

In [13]:
len(movies)

250

In [14]:
t=movies[0:100]

In [15]:
rating=[]

for rate in be.find_all('td',class_='ratingColumn imdbRating'):
    rating.append(rate.text.replace('\n',' '))

In [16]:
#100 movies
s=rating[0:100]

In [17]:
da=pd.DataFrame({'Name and Year':t,'IMDB Rating':s})

In [18]:
da

Name and Year IMDB Rating
0              1.      The Shawshank Redemption(1994)        9.2 
1                         2.      The Godfather(1972)        9.1 
2                3.      The Godfather: Part II(1974)        9.0 
3                       4.      The Dark Knight(2008)        9.0 
4                          5.      12 Angry Men(1957)        8.9 
..                                                ...         ...
95                 96.      Requiem for a Dream(2000)        8.3 
96                 97.      Singin' in the Rain(1952)        8.3 
97                  98.      North by Northwest(1959)        8.3 
98        99.      Eternal Sunshine of the Spotles...        8.3 
99                100.      Ladri di biciclette(1948)        8.3 

[100 rows x 2 columns]

# Imdb Top rated Indian Movies

In [19]:
top=requests.get('https://www.imdb.com/india/top-rated-indian-movies/')

In [20]:
top

<Response [200]>

In [21]:
bu=bs(top.content)

In [22]:
ind=[]

for h in bu.find_all('td',class_='titleColumn'):
    ind.append(h.text.replace('\n',''))
#listing 100 movies
top_rated=ind[0:100]

In [23]:
#rating
star=[]

for s in bu.find_all('td',class_='ratingColumn imdbRating'):
    star.append(s.text.replace('\n',''))
x=star[0:100]

In [24]:
#dataframe
indian=pd.DataFrame({'Name&Year':top_rated,'Rating':x})

In [25]:
indian

Name&Year Rating
0                       1.      Sardar Udham(2021)    8.6
1                            2.      Nayakan(1987)    8.5
2                         3.      Anbe Sivam(2003)    8.5
3                  4.      Pariyerum Perumal(2018)    8.5
4                  5.      C/o Kancharapalem(2018)    8.5
..                                             ...    ...
95                             96.      Roja(1992)    8.1
96                   97.      Dil Chahta Hai(2001)    8.1
97                  98.      Rang De Basanti(2006)    8.1
98                  99.      OMG: Oh My God!(2012)    8.1
99        100.      Uri: The Surgical Strike(2019)    8.1

[100 rows x 2 columns]

# Cricket Rankings

Team Rankings

In [26]:
icc=requests.get('https://www.icc-cricket.com/rankings/mens/team-rankings/odi')

In [27]:
icc

<Response [200]>

In [28]:
soup=bs(icc.content)

In [29]:
#teams names
rankings=[]

for t in soup.find_all('span',class_='u-hide-phablet'):
    rankings.append(t.text.replace(',',' '))
    
team=rankings[0:10]

In [30]:
team

['New Zealand',
 'England',
 'Australia',
 'India',
 'South Africa',
 'Pakistan',
 'Bangladesh',
 'West Indies',
 'Sri Lanka',
 'Afghanistan']

In [31]:
#printing first team's match
first_match=soup.find('td',class_='rankings-block__banner--matches')
first_match.text

'17'

In [32]:
#printing first team's point
first_point=soup.find('td',class_='rankings-block__banner--points')
first_point.text

'2,054'

In [33]:
rankings=[]

for match in soup.find_all('td',class_='table-body__cell u-center-text'):
    rankings.append(match.text)
    
#inserting first number of matches to the 0th index   
rankings.insert(0,first_match.text.replace(',',' '))

#inserting the first point to the 1st index
rankings.insert(1,first_point.text)

In [34]:
matches=rankings[0:20:2]
matches

['17', '32', '28', '32', '25', '27', '30', '30', '32', '17']

In [35]:
points=rankings[1:20:2]
points

['2,054',
 '3,793',
 '3,244',
 '3,624',
 '2,459',
 '2,524',
 '2,740',
 '2,523',
 '2,657',
 '1,054']

In [36]:
first_rate=soup.find('td',class_='rankings-block__banner--rating u-text-right')
r=first_rate.text.replace('\n',' ')
lt=r.lstrip()
rt=lt.rstrip()
rt

'121'

In [37]:
ratings=[]

for rate in soup.find_all('td',class_='table-body__cell u-text-right rating'):
    ratings.append(rate.text.replace('\n',' '))
    
 #inserting first rating to 0th index   
ratings.insert(0,rt)

In [38]:
#printing upto 10
m=ratings[0:10]
m

['121', '119', '116', '113', '98', '93', '91', '84', '83', '62']

In [39]:
print(len(matches))
print(len(points))
print(len(m))
print(len(team))

10
10
10
10


In [40]:
#printing in dataframe
rank=pd.DataFrame({'Team':team,'Matches':matches,'Points':points,'Rating':m})

In [41]:
rank

Team Matches Points Rating
0   New Zealand      17  2,054    121
1       England      32  3,793    119
2     Australia      28  3,244    116
3         India      32  3,624    113
4  South Africa      25  2,459     98
5      Pakistan      27  2,524     93
6    Bangladesh      30  2,740     91
7   West Indies      30  2,523     84
8     Sri Lanka      32  2,657     83
9   Afghanistan      17  1,054     62

Batsman rankings

In [42]:
odi=requests.get('https://www.icc-cricket.com/rankings/mens/player-rankings/odi/batting')

In [43]:
odi

<Response [200]>

In [44]:
soup=bs(odi.content)

In [45]:
#first name
name=soup.find('div',class_='rankings-block__banner--name-large')
name.text

'Babar Azam'

In [46]:
#first team
other=soup.find('div',class_='rankings-block__banner--nationality')
ot=other.text.replace('\n','')
ot

'PAK                    '

In [47]:
#first batsman's rate
rate=soup.find('div',class_='rankings-block__banner--rating')
ty=rate.text
ty

'873'

In [48]:
men=[]

for i in soup.find_all('td',class_='table-body__cell rankings-table__name name'):
    men.append(i.text.replace('\n',''))
    
n=men[0:9]
#inserting first batsman name into 0th index
n.insert(0,name.text)
n

['Babar Azam',
 'Virat Kohli',
 'Rohit Sharma',
 'Ross Taylor',
 'Aaron Finch',
 'Jonny Bairstow',
 'David Warner',
 'Shai Hope',
 'Kane Williamson',
 'Quinton de Kock']

In [49]:
teams=[]

for val in soup.find_all('span',class_='table-body__logo-text'):
    teams.append(val.text.replace('\n',''))
y=teams[0:9]
y.insert(0,ot)
y

['PAK                    ',
 'IND',
 'IND',
 'NZ',
 'AUS',
 'ENG',
 'AUS',
 'WI',
 'NZ',
 'SA']

In [50]:
four=[]

for g in soup.find_all('td',class_='table-body__cell rating'):
    four.append(g.text)
#inserting first batsman's rate    
four.insert(0,rate.text)

batsman=four[0:10]
batsman

['873', '844', '813', '801', '779', '775', '762', '758', '754', '747']

In [51]:
#dataframe
bat=pd.DataFrame({'Batsman':n,'Team':y,'Rating':batsman})

In [52]:
bat

Batsman                     Team Rating
0       Babar Azam  PAK                        873
1      Virat Kohli                      IND    844
2     Rohit Sharma                      IND    813
3      Ross Taylor                       NZ    801
4      Aaron Finch                      AUS    779
5   Jonny Bairstow                      ENG    775
6     David Warner                      AUS    762
7        Shai Hope                       WI    758
8  Kane Williamson                       NZ    754
9  Quinton de Kock                       SA    747

Bowlers Rankings

In [53]:
od=requests.get('https://www.icc-cricket.com/rankings/mens/player-rankings/odi/bowling')

In [54]:
od

<Response [200]>

In [55]:
sp=bs(od.content)

In [56]:
#name of first bowler
nt=sp.find('div',class_='rankings-block__banner--name-large')
nt.text

'Trent Boult'

In [57]:
#name of the team
tm=sp.find('div',class_='rankings-block__banner--nationality')
nz=tm.text.replace('\n','')
nz

'NZ                    '

In [58]:
#first rating
tr=sp.find('div',class_='rankings-block__banner--rating')
tr.text

'737'

In [59]:
name=[]

for nm in sp.find_all('td',class_='table-body__cell rankings-table__name name'):
    name.append(nm.text.replace('\n',''))
#inserting first bowler's name into 0th index
name.insert(0,nt.text)
nam=name[0:10]
nam

['Trent Boult',
 'Josh Hazlewood',
 'Mujeeb Ur Rahman',
 'Chris Woakes',
 'Mehedi Hasan',
 'Matt Henry',
 'Jasprit Bumrah',
 'Mitchell Starc',
 'Shakib Al Hasan',
 'Kagiso Rabada']

In [60]:
folks=[]

for j in sp.find_all('span',class_='table-body__logo-text'):
    folks.append(j.text)
#adding first team name
folks.insert(0,nz)
team=folks[0:10]
team

['NZ                    ',
 'AUS',
 'AFG',
 'ENG',
 'BAN',
 'NZ',
 'IND',
 'AUS',
 'BAN',
 'SA']

In [61]:
trend=[]
for y in sp.find_all('td',class_='table-body__cell rating'):
    trend.append(y.text)
#inserting first rate
trend.insert(0,tr.text)
rat=trend[0:10]
rat

['737', '709', '708', '700', '692', '691', '679', '652', '650', '646']

In [62]:
#dataframe
bowler=pd.DataFrame({'Bowler':nam,'Team':team,'Rating':rat})

In [63]:
bowler

Bowler                    Team Rating
0       Trent Boult  NZ                        737
1    Josh Hazlewood                     AUS    709
2  Mujeeb Ur Rahman                     AFG    708
3      Chris Woakes                     ENG    700
4      Mehedi Hasan                     BAN    692
5        Matt Henry                      NZ    691
6    Jasprit Bumrah                     IND    679
7    Mitchell Starc                     AUS    652
8   Shakib Al Hasan                     BAN    650
9     Kagiso Rabada                      SA    646

# Women's Cricket Rankings

Team rankings

In [64]:
get_data=requests.get('https://www.icc-cricket.com/rankings/womens/team-rankings/odi')

In [65]:
get_data

<Response [200]>

In [66]:
data=bs(get_data.content)

In [67]:
#first team name
first_pos=data.find('span',class_='u-hide-phablet')
t=first_pos.text
t

'Australia'

In [68]:
#first team's matches
first_mat=data.find('td',class_='rankings-block__banner--matches')
x=first_mat.text
x

'21'

In [69]:
#first team's points
first_poi=data.find('td',class_='rankings-block__banner--points')
p=first_poi.text
p

'3,379'

In [70]:
#first team's rating
first_rate=data.find('td',class_='rankings-block__banner--rating u-text-right')
f=first_rate.text.replace('\n','')
g=f.lstrip()
g

'161                            '

In [71]:
#teams names
wom=[]

for y in data.find_all('span',class_='u-hide-phablet'):
    wom.append(y.text)
tea=wom[0:10]
tea

['Australia',
 'England',
 'South Africa',
 'India',
 'New Zealand',
 'West Indies',
 'Pakistan',
 'Bangladesh',
 'Sri Lanka',
 'Ireland']

In [72]:
#number of matches and points
mat=[]

for m in data.find_all('td',class_='table-body__cell u-center-text'):
    mat.append(m.text)
#inserting first team's match   
mat.insert(0,x)
mat.insert(1,p)
mat

['21',
 '3,379',
 '25',
 '2,983',
 '29',
 '3,390',
 '26',
 '2,934',
 '26',
 '2,392',
 '22',
 '1,872',
 '20',
 '1,496',
 '5',
 '306',
 '11',
 '519',
 '2',
 '25']

In [73]:
#taking the team's number of matches
matches=mat[0:20:2]
matches

['21', '25', '29', '26', '26', '22', '20', '5', '11', '2']

In [74]:
#taking the points of the team
point=mat[1:20:2]
point

['3,379',
 '2,983',
 '3,390',
 '2,934',
 '2,392',
 '1,872',
 '1,496',
 '306',
 '519',
 '25']

In [75]:
#team rating
ex=[]

for c in data.find_all('td',class_='table-body__cell u-text-right rating'):
    ex.append(c.text)
#inserting first team's rating    
ex.insert(0,g)
ex

['161                            ',
 '119',
 '117',
 '113',
 '92',
 '85',
 '75',
 '61',
 '47',
 '13']

In [76]:
#dataframe
trank=pd.DataFrame({'Team':tea,'Matches':matches,'Points':point,'Rating':ex})

In [77]:
trank

Team Matches Points                           Rating
0     Australia      21  3,379  161                            
1       England      25  2,983                              119
2  South Africa      29  3,390                              117
3         India      26  2,934                              113
4   New Zealand      26  2,392                               92
5   West Indies      22  1,872                               85
6      Pakistan      20  1,496                               75
7    Bangladesh       5    306                               61
8     Sri Lanka      11    519                               47
9       Ireland       2     25                               13

Women Batting Rankings

In [78]:
post_data=requests.get('https://www.icc-cricket.com/rankings/womens/player-rankings/odi/batting')

In [79]:
post_data

<Response [200]>

In [80]:
batt=bs(post_data.content)

In [81]:
#name of first ranker
vi=batt.find('div',class_='rankings-block__banner--name-large')
fir=vi.text
fir

'Lizelle Lee'

In [82]:
#first ranker's team
vi1=batt.find('div',class_='rankings-block__banner--nationality')
one=vi1.text.replace('\n','')
one

'SA                    '

In [83]:
#first rating
air=batt.find('div',class_='rankings-block__banner--rating')
bus=air.text
bus

'761'

In [84]:
liz=[]

for u in batt.find_all('td',class_='table-body__cell rankings-table__name name'):
    liz.append(u.text.replace('\n',''))
#inserting name of first player
liz.insert(0,fir)
ran=liz[0:10]
ran

['Lizelle Lee',
 'Alyssa Healy',
 'Mithali Raj',
 'Tammy Beaumont',
 'Amy Satterthwaite',
 'Smriti Mandhana',
 'Meg Lanning',
 'Beth Mooney',
 'Heather Knight',
 'Laura Wolvaardt']

In [85]:
sa=[]

for o in batt.find_all('span',class_='table-body__logo-text'):
    sa.append(o.text)
#inserting first team    
sa.insert(0,one)
te=sa[0:10]
te

['SA                    ',
 'AUS',
 'IND',
 'ENG',
 'NZ',
 'IND',
 'AUS',
 'AUS',
 'ENG',
 'SA']

In [86]:
ra=[]

for n in batt.find_all('td',class_='table-body__cell rating'):
    ra.append(n.text)
#inserting first rating
ra.insert(0,bus)
ratings=ra[0:10]
ratings

['761', '750', '738', '728', '717', '710', '699', '690', '674', '672']

In [87]:
#dataframe
batra=pd.DataFrame({'Player':ran,'Team':te,'Rating':ratings})

In [88]:
batra

Player                    Team Rating
0        Lizelle Lee  SA                        761
1       Alyssa Healy                     AUS    750
2        Mithali Raj                     IND    738
3     Tammy Beaumont                     ENG    728
4  Amy Satterthwaite                      NZ    717
5    Smriti Mandhana                     IND    710
6        Meg Lanning                     AUS    699
7        Beth Mooney                     AUS    690
8     Heather Knight                     ENG    674
9    Laura Wolvaardt                      SA    672

Allrounder Rankings

In [89]:
pre=requests.get('https://www.icc-cricket.com/rankings/womens/player-rankings/odi/all-rounder')

In [90]:
pre

<Response [200]>

In [91]:
bun=bs(pre.content)

In [92]:
#first name
onn=bun.find('div',class_='rankings-block__banner--name-large')
i=onn.text
i

'Marizanne Kapp'

In [93]:
#first team
ft=bun.find('div',class_='rankings-block__banner--nationality')
ui=ft.text.replace('\n','')
ui

'SA                    '

In [94]:
#first rating
fr=bun.find('div',class_='rankings-block__banner--rating')
er=fr.text
er

'384'

In [95]:
naam=[]

for z in bun.find_all('td',class_='table-body__cell rankings-table__name name'):
    naam.append(z.text.replace('\n',''))
#inserting first name
naam.insert(0,i)
nm=naam[0:10]
nm

['Marizanne Kapp',
 'Natalie Sciver',
 'Ellyse Perry',
 'Stafanie Taylor',
 'Deepti Sharma',
 'Ashleigh Gardner',
 'Dane van Niekerk',
 'Jess Jonassen',
 'Katherine Brunt',
 'Jhulan Goswami']

In [96]:
fg=[]
for p in bun.find_all('span',class_='table-body__logo-text'):
    fg.append(p.text)
#inserting first team
fg.insert(0,ui)
hi=fg[0:10]
hi

['SA                    ',
 'ENG',
 'AUS',
 'WI',
 'IND',
 'AUS',
 'SA',
 'AUS',
 'ENG',
 'IND']

In [97]:
me=[]

for uh in bun.find_all('td',class_='table-body__cell rating'):
    me.append(uh.text)
me.insert(0,er)
me
we=me[0:10]
we

['384', '372', '365', '322', '299', '275', '274', '272', '272', '251']

In [98]:
#dataframe
allround=pd.DataFrame({'Player':nm,'Team':hi,'Rating':we})

In [99]:
allround

Player                    Team Rating
0    Marizanne Kapp  SA                        384
1    Natalie Sciver                     ENG    372
2      Ellyse Perry                     AUS    365
3   Stafanie Taylor                      WI    322
4     Deepti Sharma                     IND    299
5  Ashleigh Gardner                     AUS    275
6  Dane van Niekerk                      SA    274
7     Jess Jonassen                     AUS    272
8   Katherine Brunt                     ENG    272
9    Jhulan Goswami                     IND    251

# Mobile Phones under 20000

In [100]:
url=('https://www.amazon.in/s?k=mobile+phones+under+20000+rupees&crid=1C582EFIQSUFY&sprefix=mobile+phones+under+20000%2Caps%2C372&ref=nb_sb_ss_ts-doa-p_3_25')

In [101]:
amz=requests.get(url)

In [102]:
amz

<Response [503]>

As the response of server is 503 of the webpage. I can't scrape the details from this webpage.I kindly skip this question with your permission.

# House Details

In [103]:
url='https://www.nobroker.in/property/sale/bangalore/Electronic%20City?type=BHK4&searchParam=W3sibGF0IjoxMi44N%20DUyMTQ1LCJsb24iOjc3LjY2MDE2OTUsInBsYWNlSWQiOiJDaElKdy1GUWQ0cHNyanNSSGZkYXpnXzhYRW8%20iLCJwbGFjZU5hbWUiOiJFbGVjdHJvbmljIENpdHkifV0=&propertyAge=0&radius=2.0'

In [104]:
home=requests.get(url)

In [105]:
home

<Response [200]>

In [106]:
soup=bs(home.content)

In [107]:
#house title
title=[]

for t in soup.find_all('div',class_='nb__sDz3z'):
    title.append(t.text.split('|')[0])
title

['4 BHK In Independent House  For Sale  In Hebbagodi Independent House, Bangalore - Hosur Road, Near  National Public SchoolExplore Nearby2021-10-314 BHK In Independent House  For Sale  In HebbagodiIndependent House, Bangalore - Hosur Road, Near  National Public School4 BHK In Independent House  For Sale  In HebbagodiHebbagodibangalore4 BHK In Independent House  For Sale  In HebbagodiHebbagodibangaloreREADY',
 '4 BHK Apartment  For Sale  In Nisarga Residency In Electronic City Phase Ii Nisarga Residency\xa0 Near Thali Resturant, Ananth Nagar, Electronic City Phase II, Bangalore, Karnataka, INDIA.Explore Nearby2021-08-294 BHK Flat  For Sale  In Nisarga Residency In Electronic City Phase IiNear Thali Resturant, Ananth Nagar, Electronic City Phase II, Bangalore, Karnataka, INDIA.4 BHK Flat  For Sale  In Nisarga Residency In Electronic City Phase IiElectronic City Phase Iibangalore4 BHK Flat  For Sale  In Nisarga Residency In Electronic City Phase IiElectronic City Phase IibangaloreREADY',

In [108]:
#location
loca=[]

for lo in soup.find_all('div',class_='nb__2CMjv'):
    loca.append(lo.text)
loca

['Independent House, Bangalore - Hosur Road, Near  National Public School',
 'Nisarga Residency\xa0 Near Thali Resturant, Ananth Nagar, Electronic City Phase II, Bangalore, Karnataka, INDIA.',
 'Sobha Silicon Oasis Naganathapura, Rayasandra Bengaluru, Karnataka 560100 India',
 'Daadys Garden\xa0 Kammasandra Rd, Kammasandra, Electronic City, Bengaluru, Karnataka 560100, India',
 'Standalone Building, 16th Cross Road Neeladri Nagar, near by brand factory',
 'Standalone Building, 11th cross.anjanadri lay out',
 'Independent House, surya nagar face 1',
 'Independent House, Hosur Rd,Near Infosys Limited',
 'Gopalan Gardenia\xa0 Gopalan gardenia, Veerasandra Main Rd, Veer Sandra, Electronic City, Bengaluru, Karnataka 560100, India',
 '6th Cross']

In [109]:
#area
area=[]

for ar in soup.find_all('div',class_='nb__3oNyC'):
    area.append(ar.text)
    
area

['1,800 sqft',
 '2,000 sqft',
 '1,879 sqft',
 '2,600 sqft',
 '2,000 sqft',
 '3,000 sqft',
 '3,000 sqft',
 '1,200 sqft',
 '2,650 sqft',
 '3,100 sqft']

In [110]:
#emi
e=[]

for em in soup.find_all('div',class_='font-semi-bold heading-6'):
    e.append(em.text)
emi=e[1:29:3]
emi

['₹77,374/Month',
 '₹45,851/Month',
 '₹9,170/Month',
 '₹85,971/Month',
 '₹39,546/Month',
 '₹71,643/Month',
 '₹1.43 Lacs/Month',
 '₹42,985/Month',
 '₹68,777/Month',
 '₹85,971/Month']

In [111]:
e

['1,800 sqft',
 '₹77,374/Month',
 '₹1.35 Crores',
 '2,000 sqft',
 '₹45,851/Month',
 '₹80 Lacs',
 '1,879 sqft',
 '₹9,170/Month',
 '₹16 Lacs',
 '2,600 sqft',
 '₹85,971/Month',
 '₹1.5 Crores',
 '2,000 sqft',
 '₹39,546/Month',
 '₹69 Lacs',
 '3,000 sqft',
 '₹71,643/Month',
 '₹1.25 Crores',
 '3,000 sqft',
 '₹1.43 Lacs/Month',
 '₹2.5 Crores',
 '1,200 sqft',
 '₹42,985/Month',
 '₹75 Lacs',
 '2,650 sqft',
 '₹68,777/Month',
 '₹1.2 Crores',
 '3,100 sqft',
 '₹85,971/Month',
 '₹1.5 Crores']

In [112]:
#price
price=e[2:31:3]
price

['₹1.35 Crores',
 '₹80 Lacs',
 '₹16 Lacs',
 '₹1.5 Crores',
 '₹69 Lacs',
 '₹1.25 Crores',
 '₹2.5 Crores',
 '₹75 Lacs',
 '₹1.2 Crores',
 '₹1.5 Crores']

In [113]:
print(len(title),len(loca),len(area),len(emi),len(price))

10 10 10 10 10


In [114]:
#dataframe
house_details=pd.DataFrame({'House title':title,'Location':loca,'Area':area,'Emi':emi,'Price':price})

In [115]:
house_details

House title  \
0  4 BHK In Independent House  For Sale  In Hebba...   
1  4 BHK Apartment  For Sale  In Nisarga Residenc...   
2  4 BHK Flat  For Sale  In Sobha Silicon Oasis  ...   
3  4 BHK For Sale  In Daadys Garden In Electronic...   
4  4 BHK Flat  For Sale  In , Electronic City Sta...   
5  4 BHK Flat  For Sale  In Hosa Road, Parappana ...   
6  4 BHK In Independent House  For Sale  In Elect...   
7  4 BHK In Independent House  For Sale  In Elect...   
8  4 BHK Apartment  For Sale  In Gopalan Gardenia...   
9  4 BHK For Sale  In Gpr Royale In Gpr Royale 6t...   

                                            Location        Area  \
0  Independent House, Bangalore - Hosur Road, Nea...  1,800 sqft   
1  Nisarga Residency  Near Thali Resturant, Anant...  2,000 sqft   
2  Sobha Silicon Oasis Naganathapura, Rayasandra ...  1,879 sqft   
3  Daadys Garden  Kammasandra Rd, Kammasandra, El...  2,600 sqft   
4  Standalone Building, 16th Cross Road Neeladri ...  2,000 sqft   
5  Standalone Building, 11th cross.anjanadri lay out  3,000 sqft   
6              Independent House, surya nagar face 1  3,000 sqft   
7   Independent House, Hosur Rd,Near Infosys Limited  1,200 sqft   
8  Gopalan Gardenia  Gopalan gardenia, Veerasandr...  2,650 sqft   
9                                          6th Cross  3,100 sqft   

                Emi         Price  
0     ₹77,374/Month  ₹1.35 Crores  
1     ₹45,851/Month      ₹80 Lacs  
2      ₹9,170/Month      ₹16 Lacs  
3     ₹85,971/Month   ₹1.5 Crores  
4     ₹39,546/Month      ₹69 Lacs  
5     ₹71,643/Month  ₹1.25 Crores  
6  ₹1.43 Lacs/Month   ₹2.5 Crores  
7     ₹42,985/Month      ₹75 Lacs  
8     ₹68,777/Month   ₹1.2 Crores  
9     ₹85,971/Month   ₹1.5 Crores

# Restaurant Details

In [116]:
rest=requests.get('https://www.dineout.co.in/delhi-restaurants/buffet-special')

In [117]:
rest

<Response [200]>

In [118]:
dine=bs(rest.content)

In [119]:
#restaurant names
hotel=[]

for d in dine.find_all('div',class_='restnt-info cursor'):
    hotel.append(d.text)
hotel

['Castle BarbequeConnaught Place, Central Delhi',
 'Jungle Jamboree3CS Mall,Lajpat Nagar - 3, South Delhi',
 'Castle BarbequePacific Mall,Tagore Garden, West Delhi',
 'Cafe KnoshThe Leela Ambience Convention Hotel,Shahdara, East Delhi',
 'The Barbeque CompanyGardens Galleria,Sector 38A, Noida',
 'India GrillHilton Garden Inn,Saket, South Delhi',
 'Delhi BarbequeTaurus Sarovar Portico,Mahipalpur, South Delhi',
 'The Monarch - Bar Be Que VillageIndirapuram Habitat Centre,Indirapuram, Ghaziabad',
 'World CafeVibe by The Lalit Traveller,Sector 35, Faridabad',
 'Indian Grill RoomSuncity Business Tower,Golf Course Road, Gurgaon',
 'Mad 4 Bar B QueSector 29, Faridabad',
 'Barbeque 29NIT, Faridabad',
 'GlasshouseDoubleTree By Hilton Gurugram Baani Square,Sector 50, Gurgaon']

In [120]:
#cuisine
cui=[]

for c in dine.find_all('span',class_='double-line-ellipsis'):
    cui.append(c.text.split('|')[1])
cui

[' Chinese, North Indian',
 ' North Indian, Barbecue, Italian, Asian',
 ' North Indian, Chinese',
 ' Multi-Cuisine, North Indian, Italian, Continental, Mediterranean',
 ' Barbecue, Chinese, Mughlai, North Indian',
 ' North Indian, Italian, Oriental ',
 ' Barbecue, North Indian',
 ' North Indian, Chinese, Fast Food',
 ' North Indian, Chinese, Continental',
 ' North Indian, Mughlai, Barbecue',
 ' North Indian, Mughlai',
 ' Barbecue, Chinese, North Indian',
 ' Multi-Cuisine, Asian, European, Italian, North Indian']

In [121]:
#location
loc=[]

for l in dine.find_all('div',class_='restnt-loc ellipsis'):
    loc.append(l.text)
loc


['Connaught Place, Central Delhi',
 '3CS Mall,Lajpat Nagar - 3, South Delhi',
 'Pacific Mall,Tagore Garden, West Delhi',
 'The Leela Ambience Convention Hotel,Shahdara, East Delhi',
 'Gardens Galleria,Sector 38A, Noida',
 'Hilton Garden Inn,Saket, South Delhi',
 'Taurus Sarovar Portico,Mahipalpur, South Delhi',
 'Indirapuram Habitat Centre,Indirapuram, Ghaziabad',
 'Vibe by The Lalit Traveller,Sector 35, Faridabad',
 'Suncity Business Tower,Golf Course Road, Gurgaon',
 'Sector 29, Faridabad',
 'NIT, Faridabad',
 'DoubleTree By Hilton Gurugram Baani Square,Sector 50, Gurgaon']

In [122]:
#first rate
fir_rate=dine.find('div',class_='restnt-rating rating-3')
o=fir_rate.text
o

'3.4'

In [123]:
#ratings

ratte=[]

for u in dine.find_all('div',class_='restnt-rating rating-4'):
    ratte.append(u.text)
#inserting first rate
ratte.insert(0,o)
ratte

['3.4',
 '3.9',
 '4',
 '4.3',
 '4.1',
 '3.9',
 '3.7',
 '3.9',
 '4.3',
 '4.3',
 '3.8',
 '4.3',
 '4']

In [124]:
#image url
img=[]

for im in dine.find_all('img',class_='no-img'):
    img.append(im['data-src'])
img
    

['https://im1.dineout.co.in/images/uploads/restaurant/sharpen/8/k/b/p86792-16062953735fbe1f4d3fb7e.jpg?tr=tr:n-medium',
 'https://im1.dineout.co.in/images/uploads/restaurant/sharpen/5/a/k/p59633-16046474755fa4fa33c0e92.jpg?tr=tr:n-medium',
 'https://im1.dineout.co.in/images/uploads/restaurant/sharpen/3/j/o/p38113-15959192065f1fcb666130c.jpg?tr=tr:n-medium',
 'https://im1.dineout.co.in/images/uploads/restaurant/sharpen/4/j/v/p406-163401663361651d79326d0.jpg?tr=tr:n-medium',
 'https://im1.dineout.co.in/images/uploads/restaurant/sharpen/7/q/d/p79307-16051787075fad15532bd7c.jpg?tr=tr:n-medium',
 'https://im1.dineout.co.in/images/uploads/restaurant/sharpen/2/v/t/p2687-1482477169585cce712b90f.jpg?tr=tr:n-medium',
 'https://im1.dineout.co.in/images/uploads/restaurant/sharpen/5/v/f/p52501-16006856545f68865616659.jpg?tr=tr:n-medium',
 'https://im1.dineout.co.in/images/uploads/restaurant/sharpen/3/n/o/p34822-15599107305cfa594a13c24.jpg?tr=tr:n-medium',
 'https://im1.dineout.co.in/images/uploads/

In [125]:
print(len(hotel),len(cui),len(loc),len(ratte),len(img))

13 13 13 13 13


In [126]:
#dataframe
details=pd.DataFrame({'Restaurant name':hotel,'Cuisine':cui,'Location':loc,'Ratings':ratte,'Image url':img})

In [127]:
details

Restaurant name  \
0       Castle BarbequeConnaught Place, Central Delhi   
1   Jungle Jamboree3CS Mall,Lajpat Nagar - 3, Sout...   
2   Castle BarbequePacific Mall,Tagore Garden, Wes...   
3   Cafe KnoshThe Leela Ambience Convention Hotel,...   
4   The Barbeque CompanyGardens Galleria,Sector 38...   
5     India GrillHilton Garden Inn,Saket, South Delhi   
6   Delhi BarbequeTaurus Sarovar Portico,Mahipalpu...   
7   The Monarch - Bar Be Que VillageIndirapuram Ha...   
8   World CafeVibe by The Lalit Traveller,Sector 3...   
9   Indian Grill RoomSuncity Business Tower,Golf C...   
10                Mad 4 Bar B QueSector 29, Faridabad   
11                          Barbeque 29NIT, Faridabad   
12  GlasshouseDoubleTree By Hilton Gurugram Baani ...   

                                              Cuisine  \
0                               Chinese, North Indian   
1              North Indian, Barbecue, Italian, Asian   
2                               North Indian, Chinese   
3    Multi-Cuisine, North Indian, Italian, Contine...   
4            Barbecue, Chinese, Mughlai, North Indian   
5                    North Indian, Italian, Oriental    
6                              Barbecue, North Indian   
7                    North Indian, Chinese, Fast Food   
8                  North Indian, Chinese, Continental   
9                     North Indian, Mughlai, Barbecue   
10                              North Indian, Mughlai   
11                    Barbecue, Chinese, North Indian   
12   Multi-Cuisine, Asian, European, Italian, Nort...   

                                             Location Ratings  \
0                      Connaught Place, Central Delhi     3.4   
1              3CS Mall,Lajpat Nagar - 3, South Delhi     3.9   
2              Pacific Mall,Tagore Garden, West Delhi       4   
3   The Leela Ambience Convention Hotel,Shahdara, ...     4.3   
4                  Gardens Galleria,Sector 38A, Noida     4.1   
5                Hilton Garden Inn,Saket, South Delhi     3.9   
6      Taurus Sarovar Portico,Mahipalpur, South Delhi     3.7   
7   Indirapuram Habitat Centre,Indirapuram, Ghaziabad     3.9   
8    Vibe by The Lalit Traveller,Sector 35, Faridabad     4.3   
9    Suncity Business Tower,Golf Course Road, Gurgaon     4.3   
10                               Sector 29, Faridabad     3.8   
11                                     NIT, Faridabad     4.3   
12  DoubleTree By Hilton Gurugram Baani Square,Sec...       4   

                                            Image url  
0   https://im1.dineout.co.in/images/uploads/resta...  
1   https://im1.dineout.co.in/images/uploads/resta...  
2   https://im1.dineout.co.in/images/uploads/resta...  
3   https://im1.dineout.co.in/images/uploads/resta...  
4   https://im1.dineout.co.in/images/uploads/resta...  
5   https://im1.dineout.co.in/images/uploads/resta...  
6   https://im1.dineout.co.in/images/uploads/resta...  
7   https://im1.dineout.co.in/images/uploads/resta...  
8   https://im1.dineout.co.in/images/uploads/resta...  
9   https://im1.dineout.co.in/images/uploads/resta...  
10  https://im1.dineout.co.in/images/uploads/resta...  
11  https://im1.dineout.co.in/images/uploads/resta...  
12  https://im1.dineout.co.in/images/uploads/resta...

# Weather Details

In [128]:
wea=requests.get('https://en.tutiempo.net/delhi.html?data=last-24-hours')

In [129]:
wea

<Response [200]>

In [130]:
time=bs(wea.content)

In [131]:
#hour
h=[]

for ho in time.find_all('td'):
    h.append(ho.text)
hour=h[18:313:6]
hour

['15:30',
 '15:00',
 '14:00',
 '13:30',
 '13:00',
 '12:30',
 '12:00',
 '11:30',
 '11:00',
 '10:30',
 '10:00',
 '09:30',
 '09:00',
 '08:30',
 '08:00',
 '07:30',
 '07:00',
 '06:30',
 '06:00',
 '05:30',
 '05:00',
 '04:30',
 '04:00',
 '03:30',
 '03:00',
 '02:30',
 '02:00',
 '01:30',
 '01:00',
 '00:30',
 '00:00',
 '23:30',
 '23:00',
 '22:30',
 '22:00',
 '21:30',
 '21:00',
 '20:30',
 '20:00',
 '19:30',
 '19:00',
 '18:30',
 '18:00',
 '17:30',
 '17:00',
 '16:30',
 '16:00',
 '15:30',
 '15:00']

In [132]:
len(hour)

49

In [133]:
#temperature

temp=[]
for t in time.find_all('td',class_='t Temp'):
    temp.append(t.text)
temp

['28°C',
 '27°C',
 '27°C',
 '27°C',
 '27°C',
 '26°C',
 '26°C',
 '24°C',
 '23°C',
 '23°C',
 '22°C',
 '21°C',
 '20°C',
 '19°C',
 '18°C',
 '18°C',
 '18°C',
 '17°C',
 '18°C',
 '18°C',
 '18°C',
 '18°C',
 '19°C',
 '19°C',
 '19°C',
 '19°C',
 '19°C',
 '19°C',
 '20°C',
 '20°C',
 '20°C',
 '21°C',
 '21°C',
 '21°C',
 '21°C',
 '22°C',
 '22°C',
 '22°C',
 '23°C',
 '24°C',
 '25°C',
 '25°C',
 '25°C',
 '25°C',
 '26°C',
 '26°C',
 '26°C',
 '27°C',
 '27°C']

In [134]:
#wind
wind=[]

for wi in time.find_all('td',class_='wind'):
    wind.append(wi.text)
wind

['11 km/h',
 '13 km/h',
 '17 km/h',
 '15 km/h',
 '15 km/h',
 '15 km/h',
 '11 km/h',
 '13 km/h',
 '9 km/h',
 '9 km/h',
 '7 km/h',
 '7 km/h',
 '7 km/h',
 '9 km/h',
 '9 km/h',
 '6 km/h',
 'Calm',
 'Calm',
 'Calm',
 'Calm',
 'Calm',
 'Calm',
 'Calm',
 'Calm',
 'Calm',
 'Calm',
 'Calm',
 'Calm',
 'Calm',
 'Calm',
 'Calm',
 'Calm',
 'Calm',
 'Calm',
 'Calm',
 'Calm',
 'Calm',
 'Calm',
 'Calm',
 '6 km/h',
 '6 km/h',
 '6 km/h',
 '6 km/h',
 '6 km/h',
 '9 km/h',
 '11 km/h',
 '11 km/h',
 '13 km/h',
 '13 km/h']

In [135]:
#weather condition
wc=[]

for w in time.find_all ('td',class_='t Temp'):

    if w.previous_sibling is not None:

        wc.append(w.previous_sibling.text)

    else:

        wc.append(' ')

wc

['Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist',
 'Mist']

In [136]:
#humidity
hum=[]

for hm in time.find_all('td',class_='hr'):
    hum.append(hm.text)
hum

['42%',
 '42%',
 '48%',
 '48%',
 '51%',
 '54%',
 '54%',
 '61%',
 '69%',
 '69%',
 '78%',
 '78%',
 '83%',
 '83%',
 '88%',
 '88%',
 '88%',
 '94%',
 '88%',
 '88%',
 '88%',
 '88%',
 '88%',
 '88%',
 '88%',
 '88%',
 '88%',
 '88%',
 '83%',
 '83%',
 '83%',
 '73%',
 '73%',
 '73%',
 '73%',
 '64%',
 '64%',
 '64%',
 '61%',
 '54%',
 '47%',
 '47%',
 '47%',
 '44%',
 '42%',
 '42%',
 '42%',
 '39%',
 '42%']

In [137]:
#pressure
press=[]

for pr in time.find_all('td',class_='prob'):
    press.append(pr.text)
press

['1014 hPa',
 '1015 hPa',
 '1015 hPa',
 '1016 hPa',
 '1016 hPa',
 '1017 hPa',
 '1017 hPa',
 '1018 hPa',
 '1018 hPa',
 '1019 hPa',
 '1019 hPa',
 '1018 hPa',
 '1018 hPa',
 '1018 hPa',
 '1018 hPa',
 '1017 hPa',
 '1017 hPa',
 '1016 hPa',
 '1016 hPa',
 '1016 hPa',
 '1015 hPa',
 '1015 hPa',
 '1015 hPa',
 '1015 hPa',
 '1015 hPa',
 '1015 hPa',
 '1015 hPa',
 '1015 hPa',
 '1015 hPa',
 '1015 hPa',
 '1015 hPa',
 '1015 hPa',
 '1015 hPa',
 '1015 hPa',
 '1015 hPa',
 '1015 hPa',
 '1015 hPa',
 '1015 hPa',
 '1015 hPa',
 '1014 hPa',
 '1014 hPa',
 '1014 hPa',
 '1013 hPa',
 '1013 hPa',
 '1013 hPa',
 '1013 hPa',
 '1013 hPa',
 '1013 hPa',
 '1013 hPa']

In [138]:
print(len(hour),len(wind),len(temp),len(hum),len(press),len(wc))

49 49 49 49 49 49


In [139]:
#dataframe
weather=pd.DataFrame({'Hour':hour,'Temperature':temp,'Wind':wind,'Weather Condition':wc,'Humidity':hum,'Pressure':press})

In [140]:
weather

Hour Temperature     Wind Weather Condition Humidity  Pressure
0   15:30        28°C  11 km/h              Mist      42%  1014 hPa
1   15:00        27°C  13 km/h              Mist      42%  1015 hPa
2   14:00        27°C  17 km/h              Mist      48%  1015 hPa
3   13:30        27°C  15 km/h              Mist      48%  1016 hPa
4   13:00        27°C  15 km/h              Mist      51%  1016 hPa
5   12:30        26°C  15 km/h              Mist      54%  1017 hPa
6   12:00        26°C  11 km/h              Mist      54%  1017 hPa
7   11:30        24°C  13 km/h              Mist      61%  1018 hPa
8   11:00        23°C   9 km/h              Mist      69%  1018 hPa
9   10:30        23°C   9 km/h              Mist      69%  1019 hPa
10  10:00        22°C   7 km/h              Mist      78%  1019 hPa
11  09:30        21°C   7 km/h              Mist      78%  1018 hPa
12  09:00        20°C   7 km/h              Mist      83%  1018 hPa
13  08:30        19°C   9 km/h              Mist      83%  1018 hPa
14  08:00        18°C   9 km/h              Mist      88%  1018 hPa
15  07:30        18°C   6 km/h              Mist      88%  1017 hPa
16  07:00        18°C     Calm              Mist      88%  1017 hPa
17  06:30        17°C     Calm              Mist      94%  1016 hPa
18  06:00        18°C     Calm              Mist      88%  1016 hPa
19  05:30        18°C     Calm              Mist      88%  1016 hPa
20  05:00        18°C     Calm              Mist      88%  1015 hPa
21  04:30        18°C     Calm              Mist      88%  1015 hPa
22  04:00        19°C     Calm              Mist      88%  1015 hPa
23  03:30        19°C     Calm              Mist      88%  1015 hPa
24  03:00        19°C     Calm              Mist      88%  1015 hPa
25  02:30        19°C     Calm              Mist      88%  1015 hPa
26  02:00        19°C     Calm              Mist      88%  1015 hPa
27  01:30        19°C     Calm              Mist      88%  1015 hPa
28  01:00        20°C     Calm              Mist      83%  1015 hPa
29  00:30        20°C     Calm              Mist      83%  1015 hPa
30  00:00        20°C     Calm              Mist      83%  1015 hPa
31  23:30        21°C     Calm              Mist      73%  1015 hPa
32  23:00        21°C     Calm              Mist      73%  1015 hPa
33  22:30        21°C     Calm              Mist      73%  1015 hPa
34  22:00        21°C     Calm              Mist      73%  1015 hPa
35  21:30        22°C     Calm              Mist      64%  1015 hPa
36  21:00        22°C     Calm              Mist      64%  1015 hPa
37  20:30        22°C     Calm              Mist      64%  1015 hPa
38  20:00        23°C     Calm              Mist      61%  1015 hPa
39  19:30        24°C   6 km/h              Mist      54%  1014 hPa
40  19:00        25°C   6 km/h              Mist      47%  1014 hPa
41  18:30        25°C   6 km/h              Mist      47%  1014 hPa
42  18:00        25°C   6 km/h              Mist      47%  1013 hPa
43  17:30        25°C   6 km/h              Mist      44%  1013 hPa
44  17:00        26°C   9 km/h              Mist      42%  1013 hPa
45  16:30        26°C  11 km/h              Mist      42%  1013 hPa
46  16:00        26°C  11 km/h              Mist      42%  1013 hPa
47  15:30        27°C  13 km/h              Mist      39%  1013 hPa
48  15:00        27°C  13 km/h              Mist      42%  1013 hPa

# Monuments

In [141]:
monu=requests.get('https://www.puredestinations.co.uk/top-10-famous-monuments-to-visit-in-india/')

In [142]:
monu

<Response [200]>

In [143]:
taj=bs(monu.content)

In [144]:
#monument names
names=[]
for mn in taj.find_all('strong'):
    names.append(mn.text)
names.pop(10)
len(names)

10

In [145]:
names

['Taj Mahal, Agra',
 'Golden Temple (Harmandir Sahib), Amritsar ',
 'Meenakshi Temple, Madurai',
 'Mysore Palace, Mysore',
 'Gateway of India, Mumbai',
 'Red Fort, New Delhi',
 'Hawa Mahal, Jaipur',
 'Qutub Minar, New Delhi',
 'Sanchi Stupa, Sanchi',
 'Charminar, Hyderabad']

In [146]:
#monument description
desc=[]

for de in taj.find_all('p'):
    desc.append(de.text.replace(',',''))
description=desc[5:33:3]
description

['Enlisted in the Seven Wonders of the World The Taj Mahal is one of the most beautiful and famous buildings located in the city of Agra. This white marble monument was built by a Mughal Emperor called Shahajahan in memory of his beloved wife. Due to its amazing architecture and the history behind it this world heritage site has become very popular to visit by all travellers and romantics from all over the world.',
 'The holiest shrine and pilgrimage place located in Amritsar is The Golden Temple known as the Harmandir Sahib. This is the most famous and sacred Sikh Gurdwara in Punjab India adorned with rich history and gold gilded exterior. If you are interested in culture and history be sure to visit this popular attraction in India.',
 'Meenakshi Temple is situated on the Southern banks of Vaigai River in the temple city Madurai. This temple is dedicated to Parvati and her consort Shiva and is visited by most Hindu and Tamil devotees and architectural lovers throughout the world. It 

In [147]:
#image url
img = []

for i in taj.find_all('img'):
    img.append(i['src'])
images=img[6:-5:2]
images

['http://www.puredestinations.co.uk/wp-content/uploads/2016/11/TAJ-MAHAL-PD-BLOG.jpg',
 'http://www.puredestinations.co.uk/wp-content/uploads/2016/11/Golden-Temple-PD-BLOG.jpg',
 'http://www.puredestinations.co.uk/wp-content/uploads/2016/11/meenakshi-temple-PD-BLOG.jpg',
 'http://www.puredestinations.co.uk/wp-content/uploads/2016/11/Mysore-palace-PD-BLOG.jpg',
 'http://www.puredestinations.co.uk/wp-content/uploads/2016/11/gateway-to-india-PD-BLOG.jpg',
 'http://www.puredestinations.co.uk/wp-content/uploads/2016/11/red-fort-PD-BLOG.jpg',
 'http://www.puredestinations.co.uk/wp-content/uploads/2016/11/HAWA-MAHAL-PD-BLOG.jpg',
 'http://www.puredestinations.co.uk/wp-content/uploads/2016/11/qutub-minar-PD-BLOG.jpg',
 'http://www.puredestinations.co.uk/wp-content/uploads/2016/11/Sanchi-Stupi-PD-BLOG.jpg',
 'http://www.puredestinations.co.uk/wp-content/uploads/2016/11/Charminar-PD-BLOG-1.jpg']

In [148]:
print(len(names),len(description),len(images))

10 10 10


In [149]:
#dataframe
monument=pd.DataFrame({'Monument Name':names,'Monument Description':description,'Image Url':images})

In [150]:
monument

Monument Name  \
0                             Taj Mahal, Agra   
1  Golden Temple (Harmandir Sahib), Amritsar    
2                   Meenakshi Temple, Madurai   
3                       Mysore Palace, Mysore   
4                    Gateway of India, Mumbai   
5                         Red Fort, New Delhi   
6                          Hawa Mahal, Jaipur   
7                      Qutub Minar, New Delhi   
8                        Sanchi Stupa, Sanchi   
9                        Charminar, Hyderabad   

                                Monument Description  \
0  Enlisted in the Seven Wonders of the World The...   
1  The holiest shrine and pilgrimage place locate...   
2  Meenakshi Temple is situated on the Southern b...   
3  The Mysore Palace is a famous historical monum...   
4  Even though Mumbai is famous for its Bollywood...   
5  Declared as the UNESCO’s World Heritage Site R...   
6  Explore a blend of beauty and Rajasthan cultur...   
7  Discover one of the tallest towers in the worl...   
8  The beautiful and massive dome Sanchi Stupa al...   
9  No visit to Hyderabad should be complete witho...   

                                           Image Url  
0  http://www.puredestinations.co.uk/wp-content/u...  
1  http://www.puredestinations.co.uk/wp-content/u...  
2  http://www.puredestinations.co.uk/wp-content/u...  
3  http://www.puredestinations.co.uk/wp-content/u...  
4  http://www.puredestinations.co.uk/wp-content/u...  
5  http://www.puredestinations.co.uk/wp-content/u...  
6  http://www.puredestinations.co.uk/wp-content/u...  
7  http://www.puredestinations.co.uk/wp-content/u...  
8  http://www.puredestinations.co.uk/wp-content/u...  
9  http://www.puredestinations.co.uk/wp-content/u...